apps/dashboard/config/configuration_singleton.rb

```diff
@@ -438,6 +438,16 @@ def rails_env_production?
     rails_env == 'production'
   end
 
+  def shared_projects_root
+    # This environment varible will support ':' colon separated paths
+    paths = (ENV['OOD_SHARED_PROJECT_PATH'] || '').split(":")
```

Comment:
We need to pull any configuration from the `configuration_singleton`.  You can see there how other path-like configuration variables are extracted.

Reply:
Ohh I see - `ENV.fetch("OOD_QUOTA_PATH", "").strip.split(/:(?!\/\/)/)` so we don't split `http://example.com` kind of path

Reply:
No I just mean that all our configurations from from this one class. It's so we don't have various environment variables all over the place, all configs are located in 1 spot.

https://github.com/OSC/ondemand/blob/b21a5971e1eb44681a5468406d0ccf8cc238eec3/apps/dashboard/config/configuration_singleton.rb#L374

Reply:
OK - I am not thinking with the smart part of my brain. This is and always was a part of the configuration_singleton I just was unable to read it correctly.